app/Services/Auth/EmailService.php

```diff
@@ -36,7 +36,8 @@ public function sendToEmail(User $user)
         }
 
         $verifyUrl = $this->generateSignedUrl($user, 'email');
-        $this->send($user->email, $user->name, $verifyUrl, (bool) $user->is_signed_up);
+        $is_edit = !empty($user->signed_up_at);
```

Comment:
せっかくなので、これのゲッター作ろう！

User.php に、こういうメソッド作るとよさそう

```php
public function getIsSignedUpAttribute()
{
    return !empty($user->signed_up_at);
}
```

使い方 : 

```suggestion
        $is_edit = $user->is_signed_up;
```

Reply:
修正しました！